agir/notifications/components/push/subscriptions.js

```diff
@@ -92,15 +101,24 @@ const useWebPush = () => {
       serviceWorker = window.AgirSW.active;
```

Comment:
Il faudrait pas vérifier qu'il y en a déjà un active et prendre celui-ci plutôt que le waiting ? S'il y en a déjà un active le waiting peut rester waiting assez longtemps.

Reply:
Je me disais que c'était plus sûr, parce que le waiting va remplacer celui qui est actif à un moment.
Je me suis inspiré de cette question sur StackOverflow : https://stackoverflow.com/questions/39624676/uncaught-in-promise-domexception-subscription-failed-no-active-service-work/39762614#39762614